tests/zquantum/qaoa/estimators/cvar_estimator_test.py

```diff
@@ -87,3 +93,26 @@ def test_cvar_estimator_returns_correct_values(self, estimator, backend, operato
         # Then
         assert len(expectation_values) == len(estimation_tasks)
         assert expectation_values[0].values == pytest.approx(target_value)
+
+    def test_cvar_estimator_returns_correct_values_without_sampling(
+        self, estimator, backend, operator
+    ):
+        # Given
+        estimation_tasks = [
+            EstimationTask(operator, Circuit([H(0)]), number_of_shots=None)
+        ]
+        if estimator.alpha <= 0.5:
+            target_value = -1
+        else:
+            target_value = (-1 * 0.5 + 1 * (estimator.alpha - 0.5)) / estimator.alpha
+
+        # When
+        expectation_values = estimator(
+            backend=backend,
+            estimation_tasks=estimation_tasks,
+            use_exact_expectation_values=True,
+        )
+
+        # Then
+        assert len(expectation_values) == len(estimation_tasks)
+        assert expectation_values[0].values == pytest.approx(target_value)
```

Comment:
Actually, I think that for calculating with exact expectation values you don't even need `pytest.approx`.